Acknowledge the existence of 3.10

greg0ire opened the pull request.

greg0ire committed:
Acknowledge the existence of 3.10

derrabus commented:
Damn, I thought I could hide the branch in plain sight. 😅

Please, let's maintain this file on 3.9.x and merge it up. That makes it easier for me. Also, the README needs to be updated.

greg0ire committed:
Use a badge url that displays a status

Looks like using the name is no longer a valid way to access it. Makes
sense, the filename should be more stable.
